File: StudentClass/src/com/itche/chapter03/Student.java
package com.itche.chapter03;

public class Student extends Person{
    private double math;
    private double english;
    public Student(){}
    public Student(String name,String addr){
        super(name, addr);
    }
    public Student(String name,String addr,char sex,int age,double math,double english){
        super(name, addr, sex, age);
        this.math=math;
        this.english=english;
    }
    public String getInfo(){
        return super.getInfo()+"、数学成绩："+this.math+"、英语成绩："+this.english;
    }
}

File: kt_base/src/main/kotlin/cn/itcast/chapter03/Student.kt
package cn.itcast.chapter03

class Student(val sno:String,val grade:Int,name:String,age:Int):Person(name,age) {
    constructor(name: String,age: Int):this("",0,name,age){


    }
    constructor():this("",0){

    }


    fun stuInfo() {
        println("学号是:${sno},年级是:${grade},姓名是:${name}，年龄是${age}")
    }
}